packages/output/src/page.js

```diff
@@ -78,19 +78,29 @@ function OutputPage({
   );
 
   const regularElements = otherElements.map((element) => {
```

Comment:
Regarding `OutputPage`, I was wondering whether a small regression test for `getStoryMarkup` would be useful, to verify that the passed `pages` array is not being modified in any way.

Reply:
I felt that freezing the entire object made such a test irrelevant, as it is now impossible to modify any part of the object.

Reply:
True, when generating the story output via the story editor when saving a story. But if using the `output` package in some other way then this is not the case

Reply:
Very true. I can add a test for that specifically. Freezing the object and testing that it doesn't throw an error would be a decent test, no?

Reply:
I think so, yeah